src/content-handlers/iiif/extensions/uv-openseadragon-extension/config/config.json

```diff
@@ -425,7 +425,7 @@
         "locale": "$locale",
         "navigatorEnabled": "$navigatorEnabled",
         "clickToZoomEnabled": "$clickToZoomEnabled",
-        "pagingEnabled": "$goHome",
+        "pagingEnabled": "$twoPageView",
```

Comment:
Could you also fix this in the [uv-pdf-extension configuration](https://github.com/UniversalViewer/universalviewer/blob/0f0a20edc2d3671b228760d091596a1c5ae117f2/src/content-handlers/iiif/extensions/uv-pdf-extension/config/config.json#L340)? I think we might as well fix both places. Once that change is in, I'll happily approve and merge this, since I think it's a clear bug fix with no downsides. :-)

Reply:
Done :)